src/main/java/org/springframework/data/elasticsearch/annotations/Field.java

```diff
@@ -240,4 +241,11 @@
 	 * @since 5.1
 	 */
 	boolean storeEmptyValue() default true;
+	
+	/**
+	 * overrides the mapping field type which otherwise will be taken from corresponding {@link FieldType}
```

Comment:
```suggestion
	 * overrides the field type in the mapping which otherwise will be taken from corresponding {@link FieldType}
```
Seems a little clearer for me